tests/api_data/endpoint_data.json

```diff
@@ -529,21 +529,21 @@
             "id": "benefits",
```

Comment:
Are these changes correct?

Reply:
Yep the assessment_store endpoint responds with uppercase status enums with _ afaik